code/lib/csf-tools/src/ConfigFile.ts

```diff
@@ -226,6 +254,63 @@ export class ConfigFile {
     }
   }
 
+  removeField(path: string[]) {
+    const removeProperty = (properties: t.ObjectProperty[], prop: string) => {
+      const index = properties.findIndex(
+        (p) =>
+          (t.isIdentifier(p.key) && p.key.name === prop) ||
+          (t.isStringLiteral(p.key) && p.key.value === prop)
```

Comment:
@shilman we missed the fact that keys could be string literals ( `{ 'core': {...} }` as opposed to `{ core: {...} }`) which they were. Luckily my AST-fu has 10x today so I figured it out.

I'm sure there are other edge cases, but I think this is good for now.

Reply:
Great job!! 💪